src/main/java/uk/gov/hmcts/reform/sendletter/tasks/UploadLettersTask.java

```diff
@@ -65,12 +66,18 @@ public void run() {
 
                 // remove pdf content, as it's no longer needed
                 letter.setPdf(null);
+            } catch (FtpException exception) {
+                letter.setState(LetterState.FailedToUpload);
 
+                logger.error(String.format("Exception uploading letter %s", letter.getId()), exception);
+            } catch (IOException exception) {
+                logger.error(String.format("Failed to zip document for letter %s", letter.getId()), exception);
```

Comment:
how do you know it's zipping?

Reply:
by looking into code. zipper throws IOException